backend/src/main/java/it/chalmers/gamma/bootstrap/FrontendBootstrap.java

```diff
@@ -0,0 +1,47 @@
+package it.chalmers.gamma.bootstrap;
+
+import it.chalmers.gamma.db.entity.ITClient;
+import it.chalmers.gamma.db.entity.Text;
+
+import java.time.Instant;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+class FrontendBootstrap {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FrontendBootstrap.class);
+
+    private final BootstrapConfig cfg;
+
+    private final BootstrapServiceHelper hlp;
```

Comment:
Shouldn't this depend on only `ITClientService`, seems a bit overkill with the BootstrapServiceHelper

Reply:
This can be removed entirerly since FrontendBootstrap isn't needed anymore